src/m_menu.c

```diff
@@ -3429,6 +3429,10 @@ enum {
   auto1_flash,
 };
 
```

Comment:
Now `menu_background` will have to default to `1`.

Reply:
What do you mean?

Reply:
The menu has a solid background by default, and so has the automap. However, we are asking for automap overlay here, i.e. for removal of the solid background. So this needs a different default than the menu background.

Reply:
Or, maybe we should just revert and have two string arrays: `"on", "off", "dark"` for the menu background and `"off", "on", "dark"` for the automap overlay mode. Sorry, my advice to reuse one for the other was misleading.

Reply:
Oh, I get it now. I hadn't that noticed the strings don't match the setting.